gel-cli-instance/src/instance/backup.rs

```diff
@@ -106,4 +114,6 @@ pub trait InstanceBackup {
     ) -> Operation<()>;
     /// List backups.
     fn list_backups(&self) -> Operation<Vec<Backup>>;
+    /// Get backup details by ID.
+    fn get_backup(&self, backup_id: &BackupId) -> anyhow::Result<Backup>;
```

Comment:
This should be an Operation as well.

Reply:
Also: note that we could return the full `Backup` from the `backup` method as well to make this a single operation.